src/langchain_google_cloud_sql_pg/vectorstore.py

```diff
@@ -767,7 +767,7 @@ async def amax_marginal_relevance_search_with_score_by_vector(
         fetch_k = fetch_k if fetch_k else self.fetch_k
         lambda_mult = lambda_mult if lambda_mult else self.lambda_mult
         embedding_list = [json.loads(row[self.embedding_column]) for row in results]
-        mmr_selected = maximal_marginal_relevance(
+        mmr_selected = utils._maximal_marginal_relevance(
```

Comment:
Does this help with the lint error?

```suggestion
        mmr_selected = utils.maximal_marginal_relevance(
```

Reply:
I think we also need to bump up the version in `requirements.txt` file [here](https://github.com/googleapis/langchain-google-cloud-sql-pg-python/blob/main/requirements.txt#L2)

to `langchain-core==0.2.35`.

I'm not sure why the renovate bot didn't automatically update it.

Reply:
Might be because our repos are missing the renovate config, [like this](https://github.com/googleapis/python-storage/blob/main/renovate.json). Or some other reason.

Reply:
@vishwarajanand lets file a bug for fixing that

Reply:
[b/363488088](http://b/363488088)